action.yml

```diff
@@ -0,0 +1,209 @@
+name: 'Publish Allure report from nspcc-dev projects to NeoFS'
+description: 'This action allows Allure reports of tests running in Neo SPCC repositories to be published on NeoFS.'
+author: 'Neo SPCC'
+branding:
+  icon: 'server'
+  color: 'green'
+inputs:
+  NEOFS_WALLET:
+    description: N3 wallet for NeoFS network access
+    required: true
+  NEOFS_WALLET_PASSWORD:
+    description: N3 wallet password
+    required: true
+  NEOFS_NETWORK_DOMAIN:
+    description: Rpc endpoint domain address
+    required: false
+    default: 'st1.storage.fs.neo.org'
+  NEOFS_HTTP_GATE:
+    description: HTTP Gateway domain address
+    required: false
+    default: 'http.fs.neo.org'
+  STORE_OBJECTS_CID:
+    description: Container ID for your data
+    required: true
+  PR_LIFETIME:
+    description: Expiration period for PRs in epochs
+    required: false
+    default: 0
+  MASTER_LIFETIME:
+    description: Expiration period for master branch in epochs
+    required: false
+    default: 0
+  MANUAL_RUN_LIFETIME:
+    description: Expiration period for manual runs in epochs
+    required: false
+    default: 0
+  OTHER_LIFETIME:
+    description: Expiration period for other events in epochs
+    required: false
+    default: 0
```

Comment:
What I don't like in this approach is that we have all this variables exposed from the action description. There has to be some way to expose a single `LIFETIME` variable and provide a corresponding lifetime to the action at the calling stage (before the subsecuent `using ...` instruction in the action-user.

Reply:
I think this simplifies a typical use case for test reports. So while we have a single `LIFETIME` in the base upload action, we can have this set here.